ci_build_images/debian.Dockerfile

```diff
@@ -5,7 +5,7 @@
 
 ARG BASE_IMAGE
 FROM "$BASE_IMAGE"
-ARG MARIADB_BRANCH=11.1
+ARG MARIADB_BRANCH=11.4
```

Comment:
This has impact on many different images. It should be left as a separate PR. See PR #680

Reply:
ack reverting, but hurry up on #680. The 11.1 base isn't maintained and its autobake-deb.sh is used to populate dependencies.